recipes/foxglove-schemas-protobuf/all/patches/0001-Add-cmakelists.txt-for-building-with-Conan.patch

```diff
@@ -0,0 +1,45 @@
+From ce4386a75e077adfd04be76aaa3e98f54ed8e657 Mon Sep 17 00:00:00 2001
+From: Kevin Lannen <kevin@luci.com>
+Date: Thu, 25 May 2023 15:30:05 -0600
+Subject: [PATCH 1/2] Add cmakelists.txt for building with Conan
+
+---
+ CMakeLists.txt | 25 +++++++++++++++++++++++++
+ 1 file changed, 25 insertions(+)
+ create mode 100644 CMakeLists.txt
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..c5ba3e8
+--- /dev/null
++++ b/CMakeLists.txt
```

Comment:
if this is adding a new file, I think it would make it a lot easier to maintain moving forwards if this was in the recipe folder and then copied to the src folder as part of the recipe creation process, _rather_ than a patch that gets applied to add a file. This is done in other recipes as well for similar cases